### PR TITLE
[self-evict 3] self evict on shutdown in testpop

### PR DESCRIPTION
**Pull request by mennopruijssers**
Let testpop `selfEvict` on `ctrl+c` (`SIGINT`).

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -92,6 +92,22 @@ function main(args) {
...
+            ringpop.selfEvict(function afterSelfEvict(err){
+                if (err) {
+                    console.error('Failure during selfEvict: ' + err);
+                    process.exit(1);
+                    return;
+                }
+                process.exit(0);
+            });
+        } else {
+            process.exit(1);
```
Comment (thanodnl) [inline main.js]: I think we should not exit here, or we should log a message that we received a second interrupt and therefore are aborting whatever we are doing and shut down.

What is the reasoning behing the emergency exit here?

Comment (mennopruijssers) [inline main.js]: Force kill by hitting `ctrl+c` twice.

**Comment by mennopruijssers** [comment]
I added better double interrupt handling and console logging.

**Comment by thanodnl** [comment]
LGTM

**Comment by Motiejus** [comment]
LGTM

**Pull request merged**